client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-menu.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SelectDropdown from 'components/select-dropdown';
+import PaginationFlow from './pagination-flow';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import DisconnectReasonsSuggestion from './disconnect-reasons-suggestion';
+
+class DisconnectReasonsMenu extends Component {
+	constructor( props ) {
+		super( props );
+		const initialState = {
+			childSelected: 'missingFeature',
+			compactButtons: false,
+			renderFull: false
+		};
+
+		this.state = initialState;
+		this._logReason = this._logReason.bind( this );
+	}
+
+	displayFull = ( option ) => {
+		return (
+			<DisconnectReasonsSuggestion
+				optionSelected={ option }
+				compactButtons={ this.state.compactButtons } />
+		);
+	}
+
+	_logReason = ( option ) => {
+		this.setState( {
+			childSelected: option.value,
+			renderFull: true
+		} );
+	}
+
+	render() {
+		const {
+			translate,
+			siteSlug
+				} = this.props;
```

Comment:
Nit: I'd close this declaration on the level I've opened it, which means: I'd delete those 2 tabs before the closing curly bracket.